packages/shared/src/internalApi/types.ts

```diff
@@ -0,0 +1,18 @@
+export namespace Internal {
```

Comment:
I'd suggest not to use namespaces at all, especially with generic names like this

Reply:
I'd opt in for namespace here, with a more specific name like `GradejsInternal` or `InternalApi`

Reply:
fixed in 274dcf8